app/src/main/java/org/oppia/app/HomeActivity.kt

```diff
@@ -15,18 +21,32 @@ class HomeActivity : AppCompatActivity() {
 
   override fun onStart() {
     super.onStart()
-    val userAppHistory = getUserAppHistory()
-    if (userAppHistory.alreadyOpenedApp) {
-      getWelcomeTextView().setText(R.string.welcome_back_text)
-    }
+    // TODO(BenHenning): Convert this to LiveData rather than risk an async operation completing outside of the
```

Comment:
Just wondering ... are you planning to keep to any specific line length?

Reply:
100 characters is what I typically use for Java wrapping, and we'll use the same for Kotlin until we have a specific Kotlin style guide to indicate this.